charts/kubescape-operator/templates/_helpers.tpl

```diff
@@ -0,0 +1,55 @@
+{{/* standardize cloud provider */}}
+{{- define "cloud_provider" -}}
+  {{- if contains "eks" .Capabilities.KubeVersion.GitVersion -}}
+    {{- print "eks" -}}
+  {{- else if contains "gke" .Capabilities.KubeVersion.GitVersion -}}
+    {{- print "gke" -}}
+  {{- else if contains "azmk8s.io" .Values.clusterServer -}}
+    {{- print "aks" -}}
+  {{- else -}}
+    {{- print "" -}}
+  {{- end }}
+{{- end }}
+
+{{- define "account_guid" -}}
+  {{- if .Values.kubescape.submit }}
```

Comment:
This is no longer relevant, KS will submit if it has URL and account ID, otherwise, KS wont submit